database_manager/src/blobs_manager/import.rs

```diff
@@ -0,0 +1,45 @@
+use crate::blobs_manager::{cli::ImportBlobs, ensure_node_synced};
+use eth2::{types::EthSpec, BeaconNodeHttpClient, Timeouts};
+use sensitive_url::SensitiveUrl;
+use std::time::Duration;
+use tracing::{info, warn};
+
+pub async fn import_blobs<E: EthSpec>(config: &ImportBlobs) -> Result<(), String> {
+    let beacon_node = SensitiveUrl::parse(&config.beacon_node)
+        .map_err(|e| format!("Unable to parse beacon node url: {e:?}"))?;
+
+    let client = BeaconNodeHttpClient::new(beacon_node, Timeouts::set_all(Duration::from_secs(12)));
+
+    let (_, is_synced) = ensure_node_synced(&client).await?;
+    if !is_synced {
+        if config.allow_unsynced {
+            warn!("Beacon node is not synced");
+        } else {
+            return Err("Beacon node is not synced".to_string());
+        }
+    }
+
+    let blobs_ssz = std::fs::read(&config.input_file)
+        .map_err(|e| format!("Failed to read input file: {e:?}"))?;
```

Comment:
We should add batching here so we can experiment with importing less blobs at a time, and see if that prevents overloading the BN.